recipe/meta.yaml

```
@@ -1,16 +1,16 @@
 {% set name = "cpnest" %}
-{% set version = "0.10.1" %}
+{% set version = "0.10.2" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
-  git_rev: v0.10.1
+  git_rev: v0.10.2
```

Can you set this to use the version number above to avoid duplication errors:

```suggestion
  git_rev: v{{ version }}
```